electron-build.yml

```diff
@@ -21,10 +21,14 @@ publish:
     repo: solar
 
 win:
+  # dashes are needed for electron-builder auto-update
+  artifactName: Solar-Wallet-Setup-${version}.${ext}
   # Electron builder bug: Without this line the icon gets corrupted on Windows
   icon: ./electron/build/icon.png
 
 mac:
+  # dashes are needed for electron-builder auto-update
+  artifactName: Solar-Wallet-${version}.${ext}
```

Comment:
We still need to check whether the generated macOS zipball still has the `-mac.zip` suffix. Without the `-mac` in the filename, the updater is not able to locate the zipball.